dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp

```diff
@@ -46,10 +46,10 @@
 
 </style>
 
-<!-- 
-	Uncomment this when we are ready to use the new content fields
-	<script src="/html/contenttype-fields.js"></script>
--->
+
+<!-- Uncomment this when we are ready to use the new content fields -->
```

Comment:
We need to do feature flag.

Reply:
Isn't that going to be part of another issue?